src/main/java/calculator/util/StringUtil.java

```diff
@@ -0,0 +1,27 @@
+package calculator.util;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class StringUtil {
+    private static final Pattern PATTERN = Pattern.compile("^[0-9-+/*\\s]*$");
+
+    public boolean isNullOrEmpty(String string){
+        if(string==null||string.isEmpty()||string.equals("")){
```

Comment:
string.isEmpty() 메서드가 string.equals("") 를 포함하고 있습니다!
```suggestion
	if (string == null || string.isEmpty()) {
```